app-web/src/components/Form/TopicForm.js

```diff
@@ -76,6 +76,9 @@ export const TopicForm = () => {
         ],
       },
     };
+    const xhr = new XMLHttpRequest()
```

Comment:
@patricksimonian does this look fine to make a request to our api, or would we like to use package like axios ??

Reply:
you know what lets keep this like this for now if you haven't changed it already. axios would be preferred since its more easiliy maintained.